h/api/test/storage_test.py

```diff
@@ -538,12 +538,83 @@ def annotation_data(self):
             }
         }
 
-    @pytest.fixture
-    def fetch_annotation(self, request):
-        patcher = patch('h.api.storage.fetch_annotation', autospec=True)
-        fetch_annotation = patcher.start()
-        request.addfinalizer(patcher.stop)
-        return fetch_annotation
+
+@pytest.mark.usefixtures('fetch_annotation')
+class TestDeleteAnnotationLegacy(object):
+
+    """Tests for delete_annotation() when the postgres feature flag is off."""
+
+    def test_it_only_fetches_the_legacy_annotation(self, fetch_annotation):
+        request = self.mock_request()
+
+        storage.delete_annotation(request, "test_id")
+
+        fetch_annotation.assert_called_once_with(
```

Comment:
`assert_called_once_with()` not only ensures that it fetched the annotation from Es but also that it did not fetch it from Pg.